akka-docs/src/main/paradox/scala/camel.md

```diff
@@ -333,13 +437,30 @@ akka://some-system/user/myconsumer?autoAck=false&replyTimeout=100+millis
 ```
 
 In the following example, a custom route to an actor is created, using the
-actor's path. the Akka camel package contains an implicit `toActorRouteDefinition` that allows for a route to
+actor's path. 
+
+@@@ div { .group-scala }
+
+The Akka camel package contains an implicit `toActorRouteDefinition` that allows for a route to
 reference an `ActorRef` directly as shown in the below example, The route starts from a [Jetty](http://www.eclipse.org/jetty/) endpoint and
 ends at the target actor.
 
 @@snip [CustomRoute.scala]($code$/scala/docs/camel/CustomRoute.scala) { #CustomRoute }
 
-When a message is received on the jetty endpoint, it is routed to the Responder actor, which in return replies back to the client of
+@@@
+
+@@@ div { .group-java }
+
+@@snip [Responder.java]($code$/java/jdocs/camel/Responder.java) { #CustomRoute }
```

Comment:
Same as lines from 279~, three code snippets in different files, so we cannot use tabs to separate Java/Scala, but have to enclose them in `@@@ div { .group-java }`